backend/station_manager/app/src/main/java/com/StationManager/app/controllers/HallController.java

```diff
@@ -0,0 +1,36 @@
+package com.StationManager.app.controllers;
+
+import io.javalin.http.Context;
+
+public class HallController {
+    public static void getState(Context ctx) {
+        // Implementation
+        ctx.status(200).result("Hall state information");
+    }
+
+    public static void saveState(Context ctx) {
+        // Implementation
+        ctx.status(200).result("Hall state saved successfully");
```

Comment:
returning `200` status code with empty body should be sufficient, but let it be like that for now